posts/2020/08/versioned-hdf5-performance.md

```diff
@@ -0,0 +1,138 @@
+<!--
+.. title: Performance of the Versioned HDF5 Library
+.. slug: versioned-hdf5-performance
+.. date: 2020-08-31 08:00:00 UTC-05:00
+.. author: Melissa Weber Mendonça
+.. tags: HDF5, h5py
+.. category:
+.. link:
+.. description:
+.. type: text
+-->
+
+In a [previous post](https://labs.quansight.org/blog/2020/08/introducing-versioned-hdf5/), we introduced the Versioned HDF5 library and described some of its features. In this post, we'll show some of the performance analysis we did while developing the library, hopefully making the case that reading and writing versioned HDF5 files is efficient and useful.
+
+<!-- TEASER_END -->
+
+## What are we measuring?
+
+*Performance* can mean different things for different operations. For the tests presented here, the main goals are:
+
+- To evaluate the performance of reading/writing Versioned HDF5 files and compare it with non-versioned files (that is, files where only the last version of the datasets is stored);
```

Comment:
_"*Performance* can mean different things" ... "To evaluate the performance "_ is a little odd - maybe add (size on disk, and I/O speed) in brackets?